src/functions/memo-management/drive-memos/get-memos-pdf/index.ts

```diff
@@ -1,10 +1,15 @@
 import { handlerPath } from "@libs/handler-resolver";
 
-export default {
-  handler: `${handlerPath(__dirname)}/handler.main`,
-  events: [
-    {
-      httpApi: {
+// NOTE: This lambda is marked as inactive in the OpenAPI spec (x-status: inactive).
```

Comment:
Same as previously and for all other inacvitve endpoints, please update. The comment can remain however